persistence/src/commonMain/kotlin/com/wire/kalium/persistence/backup/DatabaseImporter.kt

```diff
@@ -42,7 +45,6 @@ class DatabaseImporterImpl(private val sqlDriver: SqlDriver) : DatabaseImporter
         restoreTable("Reaction")
 
         sqlDriver.execute("""COMMIT""")
-        sqlDriver.execute("""DETACH $BACKUP_DB_ALIAS""")
```

Comment:
Do we still need to execute the DETACH quarry?

Reply:
Not only we don't need, it fails with "no such database" because we already detached by committing.